narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -80,7 +80,7 @@ function check_usage {
 	RUN_FETCH_REPOSITORY="true"
 	RUN_PUSH_TO_ORIGIN="true"
 	ZIP_LIST_RETENTION_TIME="1 min"
-	VERSION_INPUT="7.4.13"
+	VERSION_INPUT="2023.q3.0 7.4.13"
```

Comment:
I was wrong in the jira ticket, 7.3.10 should also be added.
In addition, it should be 2023.q3